simple_make.sh is missing libexif-dev dependency (Ubuntu/Debian)

##### Brief Description

OS: Debian 9 (stretch), Ubuntu 16.04
qTox version: v1.12.0-48-g6d1b1f6.  
Commit hash:   6d1b1f62aba08b84e4dddf3436a936cddea3518f
toxcore:  0.1.2
Qt:  5.5.1
Hardware:  VirtualBox

Reproducible: Always

##### Steps to reproduce

1. Run simple_make.sh

##### Observed Behavior
```
-- Checking for one of the modules 'libexif'
CMake Error at cmake/Dependencies.cmake:73 (message):
  LIBEXIF package, library or framework not found
Call Stack (most recent call first):
  cmake/Dependencies.cmake:101 (search_dependency)
  CMakeLists.txt:80 (include)


-- Configuring incomplete, errors occurred!

```

##### Expected Behavior

Should build successfully 

Fixable by installing `libexif-dev` package.

--- a/simple_make.sh
+++ b/simple_make.sh
@@ -12,6 +12,7 @@ apt_install() {
         cmake
         git
         libavdevice-dev
+        libexif-dev
         libgdk-pixbuf2.0-dev
         libglib2.0-dev
         libgtk2.0-dev

